workflow-testing/src/test/java/com/squareup/workflow1/WorkflowsLifecycleTests.kt

```diff
@@ -19,7 +20,9 @@ class WorkflowsLifecycleTests {
     RuntimeConfigOptions.RENDER_PER_ACTION,
     setOf(RENDER_ONLY_WHEN_STATE_CHANGES),
     setOf(CONFLATE_STALE_RENDERINGS),
-    setOf(CONFLATE_STALE_RENDERINGS, RENDER_ONLY_WHEN_STATE_CHANGES)
+    setOf(CONFLATE_STALE_RENDERINGS, RENDER_ONLY_WHEN_STATE_CHANGES),
+    setOf(RENDER_ONLY_WHEN_STATE_CHANGES, PARTIAL_TREE_RENDERING),
+    setOf(CONFLATE_STALE_RENDERINGS, RENDER_ONLY_WHEN_STATE_CHANGES, PARTIAL_TREE_RENDERING),
   ).asSequence()
```

Comment:
Not in scope but… why isn't this just `sequenceOf(…)`?

Reply:
i'm not sure tbh. the way it was written originally is likely just a mistake?